romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
May I suggest moving this under `github.com/romana/pkg/util/k8s/<somename>`. Since `/romana` is our CLI repo and satellites.